### ISSUE TITLE
Support for protobuf?

### ISSUE THREAD

**Issue by markus080402** [BlueMatthew/ProtobufInspector#3]
Hi!

Would it be feasible to add support for protobuf that is used by google play?

See attached screenshot.

Thanks and regards 
![unknown-17](https://user-images.githubusercontent.com/8713210/149673973-c5c0e6e7-3152-4f89-b6e7-674f20315b73.png)

**Comment by BlueMatthew** [comment]
Please check out version 1.4. I add it with hard code and will think how to put it into configuration of fiddler

**Comment by markus080402** [comment]
Cool thanks!

I will try it out, where did you put the files:
Protobuf2Json.dll
Inspectors/ProtobufInspector.dll
?

Thanks and regards

**Comment by BlueMatthew** [comment]
You may download it here:

https://github.com/BlueMatthew/ProtobufInspector/releases

**Comment by markus080402** [comment]
Seems to work, thanks for the update!

**Comment by markus080402** [comment]
Is it also possible to generate a .proto file from the decoded protobuf?

Thanks and regards

**Comment by BlueMatthew** [comment]
No, it can't gen proto file and it's not reversible. I just use the method like "protoc --decode_raw" to parse the data without proto and it's human-readable(json style). But it is not always accurate.
